server/config/firebaseInit.js

```diff
@@ -0,0 +1,15 @@
+// import admin from 'firebase-admin';
+
+const admin = require('firebase-admin');
+
+const serviceAccount = require('./firebase.json');
```

Comment:
![image](https://github.com/awctw/study-dash/assets/56493743/a4df0b0a-81c2-4007-b59c-6e59d7b608f2)

Never worked with firebase before so am not entirely sure how to get this setup. Is it intended that I create a Firebase project and create one from console.firebase? Do I need to install firebase-tools and run npm init? Any particular settings that I should be aware of?

Reply:
Also, are there any Firebase considerations for the Render deployment? We will need it to be solid for Saturday.